docs/fastdds/xml_configuration/common.rst

```diff
@@ -183,16 +183,14 @@ It controls the behavior of *Fast DDS* when the value of an instance changes bef
 communicated to some of its existing DataReaders.
 Please refer to :ref:`HistoryQosPolicyKind` for further information on HistoryQoS.
 
-.. |KEEP_LAST| replace:: |KEEP_LAST_HISTORY_QOS-api|
-.. |KEEP_ALL| replace:: |KEEP_ALL_HISTORY_QOS-api|
 
 +-------------+---------------------------------------------------------+-------------------------+--------------------+
 | Name        | Description                                             | Values                  | Default            |
 +=============+=========================================================+=========================+====================+
-| ``<kind>``  | *Fast DDS* will only attempt to keep the latest values  | |KEEP_LAST|             | |KEEP_LAST|        |
+| ``<kind>``  | *Fast DDS* will only attempt to keep the latest values  | :class:`KEEP_LAST`      | :class:`KEEP_LAST` |
```

Comment:
In general, I think it'd be better to have the same references to the API as we had. but printing the text you wrote here.